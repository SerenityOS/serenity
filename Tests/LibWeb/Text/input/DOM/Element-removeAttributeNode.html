<script src="../include.js"></script>
<script>
    test(() => {
        let e = document.createElement("div");
        e.setAttribute("foo", "bar");
        let a = e.getAttributeNode("foo");
        let removed = e.removeAttributeNode(a);
        println(a === removed);
        try {
            e.removeAttributeNode(a);
            println("FAIL");
        } catch (e) {
            println("OK: " + e);
        }
    });
</script>
