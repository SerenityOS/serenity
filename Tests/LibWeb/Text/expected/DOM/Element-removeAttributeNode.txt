true
OK: NotFoundError: Attribute not found
